src/Associations.cc

```diff
@@ -33,6 +34,8 @@ static double sqr(double x) {return x*x;}
 namespace lsst {
 namespace jointcal {
 
+static LOG_LOGGER _log = LOG_GET("jointcal.Associations");
```

Comment:
Because I'd missed the part about the anonymous namespace in the earlier discussion. Please see the latest commit: is that more like what you want?

We definitely need DM-9418 to help with this sort of question.